mock-plugin/src/test/java/io/mosip/signup/plugin/mock/service/MockIdentityVerifierPluginImplTest.java

```diff
@@ -0,0 +1,121 @@
+import io.mosip.signup.api.dto.FrameDetail;
+import io.mosip.signup.api.dto.IdentityVerificationDto;
+import io.mosip.signup.api.dto.VerificationResult;
+import io.mosip.signup.api.exception.IdentityVerifierException;
+import io.mosip.signup.api.util.VerificationStatus;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.test.util.ReflectionTestUtils;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class MockIdentityVerifierPluginImplTest {
+
+    @InjectMocks
+    private MockIdentityVerifierPluginImpl mockIdentityVerifierPlugin;
+
+    @Mock
+    private RestTemplate restTemplate;
+
+    @Mock
+    ResourceLoader resourceLoader;
+
+
+    @Mock
+    private KafkaTemplate kafkaTemplate;
+
+    ObjectMapper objectMapper;
+
+    @Before
+    public void before(){
+        objectMapper = new ObjectMapper();
+        ReflectionTestUtils.setField(mockIdentityVerifierPlugin, "objectMapper",objectMapper);
+    }
+
+
+    @Test
+    public void verify_withValidIdentityVerificationDto_thenPass() throws IdentityVerifierException, IOException {
+
+        String transactionId = "transactionId123";
+        IdentityVerificationDto identityVerificationDto = new IdentityVerificationDto();
+        identityVerificationDto.setStepCode("START");
+        List<FrameDetail> frameDetails= new ArrayList<>();
+        FrameDetail frameDetail = new FrameDetail();
+        frameDetail.setFrame("frame");
+        frameDetail.setOrder(0);
+        frameDetails.add(frameDetail);
+        identityVerificationDto.setFrames(frameDetails);
+
+        String jsonContent = "{\"scenes\":[{\"frameNumber\":0,\"stepCode\":\"START\",\"step\":{\"code\":\"liveness_check\",\"framesPerSecond\":3,\"durationInSeconds\":100,\"startupDelayInSeconds\":2,\"retryOnTimeout\":false,\"retryableErrorCodes\":[]},\"feedback\":null}],\"verificationResult\":{\"status\":\"COMPLETED\",\"verifiedClaims\":{\"fullName\":{\"trust_framework\":\"XYZ TF\",\"verification_process\":\"EKYC\",\"assurance_level\":\"Gold\",\"time\":\"34232432\"}},\"errorCode\":null}}";
+        Resource resource = Mockito.mock(Resource.class);
+        Mockito.when(resourceLoader.getResource(Mockito.anyString())).thenReturn(resource);
+        Mockito.when(resource.getInputStream()).thenReturn(new ByteArrayInputStream(jsonContent.getBytes()));
+
+        mockIdentityVerifierPlugin.verify(transactionId, identityVerificationDto);
```

Comment:
I don't see any assertion or any verification of method invocations.

Reply:
added verify